packages/api/helper/nominationGsheetToDB.js

```diff
@@ -25,6 +21,10 @@ module.exports = function gsheetToDB() {
     }
   });
 
+  async function findAllDomains() {
+    domains = await domainController.findAll();
```

Comment:
What is this supposed to do? Seems like this variable is not used.

Reply:
I went ahead and removed it Austin. It was probably pasted there in error.

Reply:
Integrated domains table.